src/styles/variables.css

```diff
@@ -92,4 +94,9 @@
   /* transition */
   --transition-hover: 200ms;
   --transition-appear: 500ms;
+
+  /* weights */
+  --black-weight: 900;
+  --bold-weight: 700;
+  --book-weight: 400;
```

Comment:
These are basically 1-1 mappings, so I don't think we need them. This was more relevant for Semantic themes, which referenced variables outside the scope of a particular theme. But in our case, we can just reference `900`, `700` etc. directly.

Reply:
We used to have them in our Storybook library, though

Reply:
That's a good point @Dart-EY but I wouldn't want to change it to something like `--book-weight: 900`. That will get really confusing. 

Personally I don't think we need them, but if you want to keep them, we can, but I think in that case, they need to be contextual. Something like...

```
--weight-body: 400;
--weight-heading: 700;
--weight-display: 900;
```

Reply:
Got it! I will keep them then, but change the naming to the above!